jflex/src/main/jflex/LexScan.flex

```diff
@@ -277,8 +277,8 @@ DottedVersion =  [1-9][0-9]*(\.[0-9]+){0,2}
   "%initthrow" {WSP}+ {NNL}*  { throw new ScannerException(file,ErrorMessages.QUIL_INITTHROW, yyline); }
   "%eofthrow"  {WSP}+ {QUIL} {WSP}*  { eofThrow = concExc(eofThrow,yytext().substring(10).trim()); }
   "%eofthrow"  {WSP}+ {NNL}*  { throw new ScannerException(file,ErrorMessages.QUIL_EOFTHROW, yyline); }
-  "%yylexthrow"{WSP}+ {QUIL} {WSP}*  { lexThrow = concExc(lexThrow,yytext().substring(12).trim()); }
-  "%throws"    {WSP}+ {QUIL} {WSP}*  { lexThrow = concExc(lexThrow,yytext().substring(8).trim()); }
+  "%yylexthrow"{WSP}+ {QUIL} {WSP}* {JavaComment}? { lexThrow.add(yytext().substring(12).trim()); }
```

Comment:
`{JavaComment}` is correct here (vs `JFlexComment`) if we include it in the generated code, because it's Java going to be code. We could try to separate the comment from the exception (and not include it in the generated code), but it's probably nice to have there and it'd also be fiddly to extract it.

Reply:
OK. Glad I  didn't miss something obvious.